modules/tinymce/src/plugins/link/main/ts/api/Settings.ts

```diff
@@ -38,12 +38,12 @@ const shouldShowLinkTitle = (editor: Editor) => editor.getParam('link_title', tr
 
 const allowUnsafeLinkTarget = (editor: Editor) => editor.getParam('allow_unsafe_link_target', false, 'boolean');
 
-const useQuickLink = function (editor: Editor) {
-  return editor.getParam('link_quicklink', false, 'boolean');
-};
+const useQuickLink = (editor: Editor) => editor.getParam('link_quicklink', false, 'boolean');
 
 const getDefaultLinkProtocol = (editor: Editor): string => editor.getParam('link_default_protocol', 'http', 'string');
 
+const getPlugins = (editor: Editor): string => editor.getParam('plugins', '', 'string');
```

Comment:
@HAFRMO this is one example where this inconsistent and shouldn't be in `api/Settings.ts` since it's not a setting used by the plugin to actually implement any functionality.

Reply:
Yeah, we should probably leave this where it was. `api/Settings.ts` is usually only plugin-specific settings that users use to configure the plugin, if that helps clarify? Anything else, even if it uses `getParam` can go wherever it's used. We've broken that rule a few times, I'm sure, but we should try to stick to it when we can.

Reply:
Well it wasn't in an Alien, which was why I put it in settings. But if we'd prefer not to do that and instead put it where it was, I'll do just that.

Reply:
Alien just means it's copied from somewhere else, or someone wasn't sure where it should be, so that's largely irrelevant.

Reply:
@HAFRMO just as a heads up, this still hasn't been resolved. Most were fixed, but some were missed.